doc/rtd/conf.py

```diff
@@ -82,3 +83,17 @@
 # Make sure the target is unique
 autosectionlabel_prefix_document = True
 autosectionlabel_maxdepth = 2
+
+# Sphinx-copybutton config options:
+notfound_urls_prefix = "/"
+notfound_body = (
+    "<h1>Page not found</h1><p>Sorry we missed you! Our docs have had a"
+    " remodel and some deprecated links have changed.</p><p>"
+    "<a href='https://canonical-cloud-init.readthedocs-hosted.com'>Back to our"
+    " homepage now hosted at"
+    " https://canonical-cloud-init.readthedocs-hosted.com</a></p>",
```

Comment:
This must be a string, otherwise an extra `("` and `")` are rendered wrapping the page.

```suggestion
    " https://canonical-cloud-init.readthedocs-hosted.com</a></p>"
```